src/io.c

```diff
@@ -2308,7 +2308,7 @@ _dispatch_operation_advise(dispatch_operation_t op, size_t chunk_size)
 		default: (void)dispatch_assume_zero(err); break;
 	}
 #else
-#error "_dispatch_operation_advise not implemented on this platform"
+	(void)err;
```

Comment:
I'm kinda torn on this.  This makes it easier to port - you know immediately to handle this.

Reply:
fadvise is just a hint, after all, so it is not an error if it is unavailable.